src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs

```diff
@@ -314,8 +300,20 @@ private async ValueTask<ReSharperStatus> IsReSharperRunningAsync(ReSharperStatus
                 await Task.Delay(TimeSpan.FromSeconds(2), cancellationToken).ConfigureAwait(false);
             }
 
-            async Task<int> QueryStatusAsync(OLECMD[] cmds)
+            async Task<int> QueryStatusAsync(uint cmdId)
             {
+                var hr = await QueryStatusAsync(cmdId).ConfigureAwait(false);
```

Comment:
Maybe I'm blind, but how does this not cause an infinite loop?

Reply:
❗️ The `ErrorHandler.Failed` check should go after the `QueryStatus` call. Also, this method should return `cmds[0].cmdf`.